projects/packages/scheduled-updates/src/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-update-schedules.php

```diff
@@ -266,6 +272,12 @@ public function update_item( $request ) {
 			return $result;
 		}
 
+		$verified_plugins = apply_filters( 'jetpack_scheduled_update_verify_plugins', $request['plugins'] );
+
+		if ( is_wp_error( $verified_plugins ) ) {
+			return $verified_plugins;
+		}
```

Comment:
If we'll keep it on the PHP side, this should be handled in a `validation_callback` for the plugins parameter, where the callback includes the contents of `verify_plugins` and `validate_plugin_param`

Reply:
One caveat of moving the checks into a `validate_callback` is that we will lose the custom error message. For some reason, the response will always being overwritten and return back as "Invalid paramter(s) plugins", and this kind of error message might be confusing users.

Reply:
It could look like this: https://github.com/Automattic/jetpack/compare/update/plugin-validation

It breaks all unit tests where we didn't bother installing the plugins we test with, though, so that would need fixing.

Reply:
Thanks for having a look! I tried the same yesterday and it still returns 400 with the same default error messages. I tried to run it with your branch and it shows the same result. Are we missing something here?

Reply:
Yeah, we'll probably have to do that. I was testing it via public-api and the `data` part was empty.

Reply:
Yeah, changing in Jetpack Bridge seems to benefit all, but will affect existing behavior. As long as we keep the error status the same, I think it'll be better to have a more precise error message output. Should we reach out to the team who owns Jetpack Bridge for a gut check?

Reply:
It's not owned by a team in particular, but all of Dotcom. I'll update the Draft PR to include the conversion callback like you suggested

Reply:
For future reference, D148154-code I had prepared to make the change on the Jetpack Bridge side